concepticondata/conceptlists/Carling-2017-112e.tsv

```diff
@@ -1,113 +1,113 @@
+Carling-2017-112e-46	46	1507	FRUIT	fruit	5046	Crops & fruit	
+Carling-2017-112e-47	47	346	BANYAN	banyan	5047	Crops & fruit	
+Carling-2017-112e-48	48	410	YAM	yam (Dioscorea)	5048	Crops & fruit	
+Carling-2017-112e-49	49	2445	PAPAYA	papaya	5049	Crops & fruit	
+Carling-2017-112e-50	50	2398	MANGO	mango	5050	Crops & fruit	
+Carling-2017-112e-51	51	1844	BOAT	boat	5051	Cultural artefacts & tools	
+Carling-2017-112e-52	52	1970	CANOE	canoe	5052	Cultural artefacts & tools	
+Carling-2017-112e-53	53			oar / paddle	5053	Cultural artefacts & tools	
+Carling-2017-112e-54	54	786	SAILCLOTH	sail	5054	Cultural artefacts & tools	
+Carling-2017-112e-55	55	1948	SAW	saw	5055	Cultural artefacts & tools	
+Carling-2017-112e-56	56	677	AXE	axe	5056	Cultural artefacts & tools	
+Carling-2017-112e-57	57	1787	LOOM	loom	5057	Cultural artefacts & tools	
+Carling-2017-112e-58	58	1462	POT	pot	5058	Cultural artefacts & tools	
+Carling-2017-112e-59	59	284	HOE	hoe	5059	Cultural artefacts & tools	
+Carling-2017-112e-60	60			Jew's harp	5060	Cultural artefacts & tools	
+Carling-2017-112e-61	61	140	PADDY FIELD	field (wet) / rice paddy	5061	Cultural artefacts & tools	
+Carling-2017-112e-62	62	2794	FIELD (UPLAND)	field (dry)	5062	Cultural artefacts & tools	
+Carling-2017-112e-63	63	224	MORTAR CRUSHER	mortar	5063	Cultural artefacts & tools	
+Carling-2017-112e-64	64	225	PESTLE	pestle	5064	Cultural artefacts & tools	
+Carling-2017-112e-65	65	1075	TRAP (PITFALL)	trap	5065	Cultural artefacts & tools	
+Carling-2017-112e-66	66			snare	5066	Cultural artefacts & tools	
+Carling-2017-112e-67	67	1218	ROPE	rope	5067	Cultural artefacts & tools	
+Carling-2017-112e-68	68	1539	BASKET	basket	5068	Cultural artefacts & tools	
+Carling-2017-112e-69	69	245	BROOM	broom	5069	Cultural artefacts & tools	
+Carling-2017-112e-70	70	195	MAT	mat	5070	Cultural artefacts & tools	
+Carling-2017-112e-71	71	1895	CLOTHES	clothing	5071	Cultural artefacts & tools	
+Carling-2017-112e-72	72	2282	SEA OR OCEAN	sea / ocean	5072	Seasons & nature	
+Carling-2017-112e-73	73			pond / lake	5073	Seasons & nature	
+Carling-2017-112e-74	74			snow / ice	5074	Seasons & nature	
+Carling-2017-112e-75	75	1342	SUMMER	summer / warm season	5075	Seasons & nature	
+Carling-2017-112e-76	76	1471	WINTER	winter / cold season	5076	Seasons & nature	
+Carling-2017-112e-77	77			monsoon season	5077	Seasons & nature	
+Carling-2017-112e-78	78	1276	AUTUMN	autumn	5078	Seasons & nature	
+Carling-2017-112e-79	79	1537	SPRINGTIME	spring	5079	Seasons & nature	
+Carling-2017-112e-80	80			iron / metal	5080	Metals & materials	
+Carling-2017-112e-81	81	759	SILVER	silver	5081	Metals & materials	
+Carling-2017-112e-82	82	1369	GOLD	gold	5082	Metals & materials	
+Carling-2017-112e-83	83			pearl	5083	Metals & materials	
+Carling-2017-112e-84	84			cowry shell	5084	Metals & materials	
+Carling-2017-112e-85	85	1803	WOOD	wood	5085	Metals & materials	
+Carling-2017-112e-86	86	857	STONE	stone	5086	Metals & materials	
+Carling-2017-112e-87	87			leather / hide	5087	Metals & materials	
```

Comment:
Leather and hide can be linked to leather.

Reply:
So leave unlinked for now and I'll revisit after we've decided on the concepts that should be added?